shub/apps/main/views/download.py

```diff
@@ -123,15 +123,15 @@ def _download_container(container, request):
         return HttpResponseForbidden()
 
     # A remove build will store a metadata image url
-    elif "image" in container.metadata:
+    if "image" in container.metadata:
```

Comment:
This needs to be elif, as there could be containers (that had a registry storage switch) that have both "image" in the container.metadata and container.image, and we most certainly don't want both to run.

Reply:
But doesn't the first `if` return? So if goes down that branch it would never reach the next `if` branch?

Reply:
Yes you’re right! Sorry I shouldn’t review when I’m tired 😴